### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by TrentonWalkup**
added the ability to make reports

### CODE REVIEW

**Comment by Gregorma** [comment]
LGTM, and I updated my scripts as well

File: demo/regressionSuite/ao-login.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var expectedTitleMyAccount = 'My Account | LegalShield Associates Only Portal'
 var expectedTitleMyBusiness = 'My Business | LegalShield Associates Only Portal'
 var expectedTitleLogOut = "You've Been Logged Out | LegalShield Associates Only Portal"
 
-who('USER', function() {
+who('As a USER on AO', function() {
 what('logging into ao succesfully', function() {
```
Comment (taylorlacy) [inline demo/regressionSuite/ao-login.js]: "succesfully" should be "successfully"

File: demo/regressionSuite/ao-login.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ var expectedTitleMyBusiness = 'My Business | LegalShield Associates Only Portal'
 var expectedTitleLogOut = "You've Been Logged Out | LegalShield Associates Only Portal"
 
 who('As a USER on AO', function() {
-what('logging into ao succesfully', function() {
+what('logging into ao successfully', function() {
```
Comment (TrentonWalkup) [inline demo/regressionSuite/ao-login.js]: @taylorlacy please review!

Comment (taylorlacy) [reply demo/regressionSuite/ao-login.js]: It's fixed! LGTM!

**Pull request merged**